src/database/initializeDatabase.mjs

```diff
@@ -5,6 +5,5 @@ const agent = superagent.agent();
 const baseUrl = process.env.BASE_URL;
 
 export const initializeDatabase = async (tableName) => {
-	console.log(baseUrl);
 	return await agent.get(`${baseUrl}/${tableName}/initializeDatabase`);
```

Comment:
Is dit niet logischer ${baseUrl}/initializeDatabase/${tableName}? Want het is anders toch ook bvb. user/userId/plant/plantId

Reply:
Ik zal er een ticketje voor aan maken :)